app/assets/javascripts/listings/components/saleStats.js.coffee

```diff
@@ -0,0 +1,29 @@
+angular.module('dahlia.components')
+.component 'saleStats',
+  templateUrl: 'listings/components/sale-stats.html'
+  bindings:
+    listing: '<'
+  require:
+    listingContainer: '^propertyCard'
+  controller: () ->
+      ctrl = @
+
+      @hasUnitsWithoutParking = (listing) ->
+        for unitSummary in (listing.unitSummaries.general || []).concat(listing.unitSummaries.reserved || [])
+          if unitSummary.minPriceWithoutParking
+            return true
+        false
```

Comment:
can't do a multiline code suggestion so i'll just put this here - suggestion for `@hasUnitsWithoutParking`:

```
@hasUnitsWithoutParking = (listing) ->
  allUnitSummaries = _.concat(listing.unitSummaries.general, listing.unitSummaries.reserved)
  _.any(allUnitSummaries, 'minPriceWithoutParking')
```

Reply:
if this is acceptable to you @akegan then you could apply the same idea to other functions in this PR that are similarly structured with for loops and property presence checking

Reply:
yep can do. Unfortunately _.any doesn't exist. re-working now to still make it better

Reply:
oh right sorry that's old lodash usage, use `_.some` now